File: 프로그래머스/1/12930. 이상한 문자 만들기/이상한 문자 만들기.py
def solution(s):
    answer = ''
    for x in s.split(' '):
        for i in range(len(x)):
            if i%2==0:
                answer+=x[i].upper()
            else:
                answer+=x[i].lower()
        answer+=' '
    return answer[:-1]

File: 프로그래머스/1/12930. 이상한 문자 만들기/이상한 문자 만들기.js
function solution(s) {
    var answer = '';
    const newStr = s.split(' ')
    for(let i=0; i<newStr.length; i++){
        for(let j=0; j<newStr[i].length; j++){
            if(j%2==0) answer+=newStr[i][j].toUpperCase()
            else answer+=newStr[i][j].toLowerCase()
        }
        answer+=' '
    }
    return answer.slice(0,answer.length-1);
}